app/const.go

```diff
@@ -25,7 +25,7 @@ const (
 	stakingMsgBeginRedelegate           = "/cosmos.staking.v1beta1.MsgBeginRedelegate"
 	stakingMsgCreateValidator           = "/cosmos.staking.v1beta1.MsgCreateValidator"
 	vestingMsgCreateVestingAccount      = "/cosmos.vesting.v1beta1.MsgCreateVestingAccount"
-	transferMsgTransfer                 = "/ibc.applications.transfer.v1.MsgTransfer"
+	ibcMsgTransfer                      = "/ibc.applications.transfer.v1.MsgTransfer"
```

Comment:
out of curiosity what is the reason for this file?

Reply:
this file is for whitelisting the allowed msg types for ica host, will add comment to this file.